KerbalStuff/thumbnail.py

```diff
@@ -51,23 +52,41 @@ def create(background_path: str, thumbnail_path: str) -> None:
     im.save(thumbnail_path, 'jpeg', quality=quality, optimize=True)
 
 
-def get_or_create(background_url: str) -> str:
+# Returns the URL for the thumbnail
+def get_or_create(mod):  # type: ignore # (would cause a circular import)
```

Comment:
```suggestion
def get_or_create(mod: 'Mod'):
```

What about a forward reference? Would that work?

Reply:
The answer was a mix of this string annotation and a conditional import behind the global `TYPE_CHECKING` variable:
https://adamj.eu/tech/2021/05/13/python-type-hints-how-to-fix-circular-imports/